src/Tickets/Seating/Frontend.php

```diff
@@ -169,13 +168,15 @@ public function print_tickets_block( $html, $file, $name, $template, $context ):
 		$prices = array_keys( $prices );
 
 		$inventory = $this->get_events_ticket_capacity_for_seating( $post_id );
-
-		$cost_range = count( $prices ) === 1 ?
-			tribe_format_currency( $prices[0], $post_id ) :
-			tribe_format_currency( min( $prices ), $post_id )
+		
+		/** @var Tribe__Tickets__Commerce__Currency $currency */
+		$currency = tribe( 'tickets.commerce.currency' );
+		$currency->get_formatted_currency_with_symbol( 0, $post_id, $provider, false );
```

Comment:
What is this line for?

Reply:
Currency could possibly be different based on the provider